bevy_kayak_renderer/src/render/unified/pipeline.rs

```diff
@@ -376,9 +379,25 @@ pub fn prepare_quads(
 
     sprite_meta.types_buffer.clear();
     // sprite_meta.types_buffer.reserve(2, &render_device);
-    let quad_type_offset = sprite_meta.types_buffer.push(QuadType { t: 0 });
-    let text_type_offset = sprite_meta.types_buffer.push(QuadType { t: 1 });
-    let image_type_offset = sprite_meta.types_buffer.push(QuadType { t: 2 });
+    let quad_type_offset = sprite_meta.types_buffer.push(QuadType {
+        t: 0,
+        _unused_1: 0,
```

Comment:
Instead of `_unused` can we please use `_padding`? Thanks!

Reply:
Changed in the latest commit :+1: